WeakAuras/Init.lua

```diff
@@ -369,13 +371,28 @@ local flavorFromTocToNumber = {
 }
 local flavor = flavorFromTocToNumber[flavorFromToc]
 
+if versionString:find("alpha") then
```

Comment:
i think this is wrong, but i'm not sure what's a good way to detect alpha builds. is --@alpha@ a thing?

Reply:
https://github.com/BigWigsMods/packager#build-type-keywords